helios-client/src/main/java/com/spotify/helios/common/JobValidator.java

```diff
@@ -150,7 +150,7 @@ public JobValidator(final boolean shouldValidateJobHash,
         errors.add("Volume path is not absolute: " + path);
         continue;
       }
-      if (!isNullOrEmpty(source) && !source.startsWith("/")) {
+      if (isNullOrEmpty(source) && !source.startsWith("/")) {
```

Comment:
Are you sure this logic wasn't correct before? Wouldn't this now throw an NPE if `source` is null?